src/main/scripts/org/reaktivity/specification/nukleus/tcp/streams/rfc793/client.sent.reset.and.end/client.rpt

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newClientAcceptRef ${nuklei:newReferenceId()} # external scope
+
+connect await ROUTED_CLIENT
+        "nukleus://tcp/streams/source"
+    option nukleus:route ${newClientAcceptRef}
+    option nukleus:window 8192
+    option nukleus:transmission "duplex"
+
+connected
+
+read "some data"
```

Comment:
There's no point reading data here, that's not what we're testing in this test, please remove

Reply:
Done.